### PR TITLE
add unit tests

### PR DESCRIPTION
**Pull request by cyber-kid**
I have added stub implementation and a unit test. Please, check.

### CODE REVIEW

**Comment by softwareengineerhub** [comment]
Please fix unit tests

**Comment by cyber-kid** [comment]
fixed

File: SampleOfApplications/src/test/java/com/app/datastructures/SimpleStackTest.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+        stack.push(item1);
+        stack.push(item2);
+
+        assertEquals("item 2", stack.pop());
+
+        stack.push(item3);
+
+        assertEquals("item 3", stack.pop());
+        assertEquals("item 1", stack.pop());
```
Comment (denis4321) [inline SampleOfApplications/src/test/java/com/app/datastructures/SimpleStackTest.java]: item 1

**Review by softwareengineerhub — APPROVED** [review]
Accept

**Comment by softwareengineerhub** [comment]
Apply changes

**Pull request merged**